test/e2e/index.ts

```diff
@@ -1298,28 +1287,3 @@ test(
     t.equal(response, "hi", "updated coordinator zomes can be called");
   })
 );
-
-test(
```

Comment:
Couldn't find a sensible way to retain this. I think if the websocket connection drops then the app needs to reconnect